### PR TITLE
Minor changes to readme and docs

### PR DESCRIPTION
**Pull request by palak2011**

### CODE REVIEW

**Comment by palak2011** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -35,13 +27,13 @@ escaped quotes (e.g., `spark.ui.port\"`) in map keys can cause serialization pro
 extra attention to make sure no offending escaping is in your `SparkAppliction` CRD objects, particularly if you use 
 Kubernetes prior to 1.10.**   
 
-## Documentation
+## Get Started
+
+Get started quickly with the Spark Operator using [Quick Start Guide](docs/quick-start-guide.md).
```
Comment (liyinan926) [inline README.md]: Add `the` before `Quick Start Guide`.

File: README.md [hunk 2]
```diff
@@ -65,7 +57,7 @@ To make such automation possible, Spark Operator uses the `SparkApplication` CRD
 [initializer](https://kubernetes.io/docs/admin/extensible-admission-controllers/#initializers). The CRD controller setups the 
 environment for an application and submits the application to run on behalf of the user, whereas the initializer handles customization 
 of the Spark Pods. It also supports running Spark applications on standard [cron](https://en.wikipedia.org/wiki/Cron) schedules using 
-the `ScheduledSparkApplication` CRD and the corresponding CRD controller.
+the `ScheduledSparkApplication` CRD and the corresponding CRD controller.y
```
Comment (liyinan926) [inline README.md]: Remove the trailing `y`.

**Comment by palak2011** [comment]
I signed it!

**Pull request merged**